app/components/autocomplete/slash_suggestion/slash_suggestion_item.tsx

```diff
@@ -10,6 +10,9 @@ import {Theme} from '@mm-redux/types/preferences';
 import TouchableWithFeedback from '@components/touchable_with_feedback';
 import {changeOpacity, makeStyleSheetFromTheme} from '@utils/theme';
 import FastImage from 'react-native-fast-image';
+import {SvgCss} from 'react-native-svg';
```

Comment:
I guess this is ok

Reply:
Since we are not using any CSS, I changed it to SvgXml